omero/sysadmins/unix/install-web/web-deployment.rst

```diff
@@ -1,32 +1,44 @@
-OMERO.web installation
-======================
+OMERO.web installation and maintenance
+======================================
 
-OMERO.web is a Python client of the OMERO platform that provides a
+OMERO.web is a Python 3 client of the OMERO platform that provides a
 web-based UI and JSON API.
-This section provides guidance on how to install and set up OMERO.web on
-any of the supported UNIX and UNIX-like platforms. Specific walkthroughs are
-provided for several systems, with detailed step-by-step instructions.
+This section provides links to detailed step-by-step walkthroughs describing how to install, customize, maintain and run OMERO.web for several systems.
 OMERO.web is installed **separately** from the OMERO.server.
 
 OMERO.web can be deployed with:
 
 -  `WSGI <https://wsgi.readthedocs.org>`_ using a WSGI capable web server
    such as
    `NGINX <https://nginx.org/>`_ and `Gunicorn <https://docs.gunicorn.org/>`_
--  the built-in Django lightweight development server (for **testing**
-   only; see the :doc:`/developers/Web/Deployment` page)
+-  the built-in Django lightweight development server. This type of deployment should only be used for **testing** purpose
+   only; see the :doc:`/developers/Web/Deployment` page.
 
 If you need help configuring your firewall rules, see
 :doc:`/sysadmins/server-security` for more details.
 
-If you have installed NGINX, OMERO can automatically generate a
+Depending upon which platform you are using, you may find a
+more specific walkthrough listed below. The guides use the example of deploying OMERO.web with
+`NGINX <https://nginx.org/>`_ and `Gunicorn <https://docs.gunicorn.org/>`_.
+OMERO can automatically generate a
 configuration file for your webserver. The location of the file will depend
-on your system, please refer to your webserver's manual. See
-:ref:`customizing_your_omero_web_installation` for additional
-customization options.
+on your system, please refer to your webserver's manual. See in the `customization your installation` section in the various walkthroughs for more options.
```

Comment:
1. ``customization your installation`` sounds wrong (missing ``of`` ?)
2. is that supposed to be a link ? Does not point anywhere.

![Screen Shot 2020-06-16 at 11 19 15](https://user-images.githubusercontent.com/2478303/84763430-1476c400-afc4-11ea-8821-ac4f9b859def.png)